feat: create pylama.yml

memorisanka opened the pull request:
Details:
- Added 'pylama.yml' file to the project root directory.
- Configured Pylama to run with specified linters and options.

Impact:
- Improved codebase quality and maintainability.
- Reduces the chances of introducing common Python coding issues.

memorisanka committed:
feat: create pylama.yml

Details:
- Added 'pylama.yml' file to the project root directory.
- Configured Pylama to run with specified linters and options.

Impact:
- Improved codebase quality and maintainability.
- Reduces the chances of introducing common Python coding issues.

jacoor commented on .github/workflows/pylama.yml:
```diff
@@ -0,0 +1,28 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```
Please bump ubuntu to new LTS

jacoor commented on .github/workflows/pylama.yml:
```diff
@@ -0,0 +1,28 @@
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```
With pylama there is no use for multiple python version. Project uses version 3.11, we don't support others so please remove other versions

jacoor reviewed (changes requested):
Small changes please. 
Good work and nice idea with multiple python versions. It might be worth with tests, but not with code quality.

memorisanka committed:
Update pylama.yml

Hi Team,
I've changed the Python version to 3.11 and the Ubuntu version to 22.04.2 (LTS). Let me know, what do you think about these changes.

jacoor commented:
Thank you, approved. Please merge when ready!
